backend/routes/user.py

```diff
@@ -25,15 +25,27 @@ def otp_generate_save(phone_number):
 def send_otp_to_phone(phone_number, otp):
     print("Sent OTP to phone successfully !!")
 
+
 @router.post("/account")
-async def create_account(user: CreateUserSchema= Body(...)):
+async def create_account(user: CreateUserSchema = Body(...)):
     user_dict = jsonable_encoder(user)
 
     user_dict.update({'is_active': False})
 
-    # check if same record exists
-    if database.user.find_one({'phone_number': user_dict['phone_number']}):
-        raise HTTPException(status_code=409, detail="Phone number already exists !")
+    # Check if the user wants to create an account for someone else
+    if user.is_created_for_someone_else:
+        # Get the name of the person for whom the account is being created
+        full_name = user.name
+
+        # Add the new user to the database with the specified information
+        user_dict.update({
+            'full_name': full_name
+        })
+
```

Comment:
Why we are saving 'full_name' field in db ? I am missing something here, can you please explain it to me. ?

Reply:
Here what I assume that if  user select checkbox  for the someone else in CreateUserSchema model we have "name" field for user.So after select checkbox 'full_name': full_name is executed for checking correct user who are going to created account that full_name was saved in db.
 As per our requirements If full_name  filed and save it in db  is not necessary  I can remove that.

Reply:
I think already existing name field suffices the requirements. There is no requirement for saving full_name for whom we are creating the account. You can remove that.

Reply:
Yes,here I removed Full_Name and  I have commited changes regarding that